### PR TITLE
Clarification of the README

### PR DESCRIPTION
**Pull request by bidoubiwa**
This is joined with the same changes in the documentation : https://github.com/meilisearch/documentation/pull/33

### CODE REVIEW

**Comment by bidoubiwa** [comment]
This is a first sugestion on the possible changes of the readme file. I would like your feedback on it @Kerollmops .

File: README.md [hunk 1]
```diff
@@ -158,6 +163,20 @@ cargo run --release --example from_file -- \
...
+## How it works
+
+MeiliSearch uses [LMDB](https://en.wikipedia.org/wiki/Lightning_Memory-Mapped_Database) as the internal key-value store. The key-value store allows us to handle updates and queries with small memory and CPU overheads. The whole ranking system is [data oriented](https://github.com/meilisearch/MeiliSearch/issues/82) and provides great performances.
+
+You can [read the deep dive](deep-dive.md) if you want more information on the engine, it describes the whole process of generating updates and handling queries or you can take a look at the [typos and ranking rules](typos-ranking-rules.md) if you want to know the default rules used to sort the documents.
+
+## Contributing
+
+We will be glad if you submit issues and pull requests. You can help to grow this project and start contributing by checking [issues tagged "good-first-issue"](https://github.com/meilisearch/MeiliSearch/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22). It is a good start!
+
```
Comment (curquiza) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1,14 +1,31 @@
-# MeiliSearch
+# MeiliSearch
```
Comment (Kerollmops) [inline README.md]:
```suggestion
# MeiliSearch
```

File: README.md [hunk 3]
```diff
@@ -1,14 +1,31 @@
-# MeiliSearch
+# MeiliSearch 
 
 [![Build Status](https://github.com/meilisearch/MeiliSearch/workflows/Cargo%20test/badge.svg)](https://github.com/meilisearch/MeiliSearch/actions)
 [![dependency status](https://deps.rs/repo/github/meilisearch/MeiliSearch/status.svg)](https://deps.rs/repo/github/meilisearch/MeiliSearch)
 [![License](https://img.shields.io/badge/license-commons%20clause-lightgrey)](https://commonsclause.com/)
 
-Ultra relevant and instant full-text search API.
+⚡ Ultra relevant and instant full-text search API. 🔍
```
Comment (Kerollmops) [inline README.md]:
```suggestion
⚡ Ultra relevant and instant full-text search API 🔍
```

File: README.md [hunk 4]
```diff
@@ -1,14 +1,31 @@
...
 
 MeiliSearch is a powerful, fast, open-source, easy to use and deploy search engine. The search and indexation are fully customizable and handles features like typo-tolerance, filters, and ranking.
 
-## Features
+## What MeiliSearch has to offer
+* Search as you type experience (answers < 50ms)
+* Full-text search
+* Typo tolerant (understands typos and spelling mistakes)
+* Support Kanji
```
Comment (Kerollmops) [inline README.md]:
```suggestion
* Supports Kanji
```

File: README.md [hunk 5]
```diff
@@ -1,14 +1,31 @@
...
+* Highly customizable
+* RESTfull API 
+
+For more [details about those features go to our documentation](https://docs.meilisearch.com/introduction/features.html).
+
+[![crates.io demo gif](misc/crates-io-demo.gif)](https://crates.meilisearch.com)
+
+> Meili helps the Rust community find crates on [crates.meilisearch.com](https://crates.meilisearch.com)
+
+## In depth features
```
Comment (Kerollmops) [inline README.md]:
```suggestion
### In depth features
```

File: README.md [hunk 6]
```diff
@@ -4,11 +4,28 @@
...
 [![License](https://img.shields.io/badge/license-commons%20clause-lightgrey)](https://commonsclause.com/)
 
-Ultra relevant and instant full-text search API.
+⚡ Ultra relevant and instant full-text search API 🔍
 
 MeiliSearch is a powerful, fast, open-source, easy to use and deploy search engine. The search and indexation are fully customizable and handles features like typo-tolerance, filters, and ranking.
 
-## Features
+## What MeiliSearch has to offer
+* Search as you type experience (answers < 50ms)
```
Comment (bidoubiwa) [inline README.md]:
```suggestion
* Search as-you-type experience (answers < 50ms)
```

File: README.md [hunk 7]
```diff
@@ -4,11 +4,28 @@
 [![dependency status](https://deps.rs/repo/github/meilisearch/MeiliSearch/status.svg)](https://deps.rs/repo/github/meilisearch/MeiliSearch)
 [![License](https://img.shields.io/badge/license-commons%20clause-lightgrey)](https://commonsclause.com/)
 
-Ultra relevant and instant full-text search API.
+⚡ Ultra relevant and instant full-text search API 🔍
 
 MeiliSearch is a powerful, fast, open-source, easy to use and deploy search engine. The search and indexation are fully customizable and handles features like typo-tolerance, filters, and ranking.
```
Comment (bidoubiwa) [inline README.md]:
```suggestion
MeiliSearch is a powerful, fast, open-source, easy to use and deploy search engine. The search and indexation are fully customizable and handles features like typo-tolerance, filters, and synonyms.
```

File: README.md [hunk 8]
```diff
@@ -158,6 +160,16 @@ cargo run --release --example from_file -- \
     id popularity adult original_title
 ```
 
+## How it works
+
+MeiliSearch uses [LMDB](https://en.wikipedia.org/wiki/Lightning_Memory-Mapped_Database) as the internal key-value store. The key-value store allows us to handle updates and queries with small memory and CPU overheads. The whole ranking system is [data oriented](https://github.com/meilisearch/MeiliSearch/issues/82) and provides great performances.
+
+You can [read the deep dive](deep-dive.md) if you want more information on the engine, it describes the whole process of generating updates and handling queries or you can take a look at the [typos and ranking rules](typos-ranking-rules.md) if you want to know the default rules used to sort the documents.
```
Comment (curquiza) [inline README.md]:
```suggestion
You can [read the deep dive](deep-dive.md) if you want more information on the engine; it describes the whole process of generating updates and handling queries. Also, you can take a look at the [typos and ranking rules](typos-ranking-rules.md) if you want to know the default rules used to sort the documents.
```

File: README.md [hunk 9]
```diff
@@ -4,11 +4,28 @@
...
-MeiliSearch is a powerful, fast, open-source, easy to use and deploy search engine. The search and indexation are fully customizable and handles features like typo-tolerance, filters, and ranking.
+MeiliSearch is a powerful, fast, open-source, easy to use and deploy search engine. The search and indexation are fully customizable and handles features like typo-tolerance, filters, and synonyms.
 
-## Features
+## What MeiliSearch has to offer
+* Search as-you-type experience (answers < 50ms)
+* Full-text search
+* Typo tolerant (understands typos and spelling mistakes)
+* Supports Kanji
+* Supports Synonym
```
Comment (Kerollmops) [inline README.md]:
```suggestion
* Supports synonyms
```

Comment (Kerollmops) [reply README.md]:
```suggestion
* Supports synonyming
```

File: README.md [hunk 10]
```diff
@@ -43,7 +47,7 @@ You will need to create your own web front display to make it pretty though.
 
 ### Deploy the Server
 
-If you have not installed Rust and its package manager `cargo` yet, go to [the installation page](https://www.rust-lang.org/tools/install).<br/>
+If you have not yet installed Rust and its package manager `cargo`, go to [the installation page](https://www.rust-lang.org/tools/install).<br/>
```
Comment (Kerollmops) [inline README.md]:
```suggestion
If you have not yet installed Rust and its package manager `cargo`, go to [the installation page](https://www.rust-lang.org/tools/install) and install it before continuing.<br/>
```

**Pull request closed without merging**